### DISCUSSION TITLE
Noah MP运行过程问题

### DISCUSSION THREAD

**Discussion by TJH521** [NCAR/noahmp#153] [Q&A]
![1728874737986](https://github.com/user-attachments/assets/15968d26-b13a-4603-acba-d2b230edb9dc)
![1728874842497](https://github.com/user-attachments/assets/045317de-6195-451f-b2f3-8a491572d9ab
运行中停止
At line 529 of file module_Noahlsm_gridded_input.f
Fortran runtime error: Index '-999' of dimension 1 of array 'nroot_table' below lower bound of 1
Error termination. Backtrace:
#0  0x4f84f4 in ???
#1  0x5269da in ???
#2  0x401f1c in ???
#3  0x14796be547b2 in ???
#4  0x401f5d in ???
#5  0xffffffffffffffff in ???

**Comment by cenlinhe** [comment]
Are you using Noah LSM or Noah-MP LSM? They are very different. I do not think we have module_Noahlsm_gridded_input.f file inside Noah-MP code.

**Reply by TJH521** [reply]
I used Mr. Niu Guoyue's model, which seems to be Noah-MP LSM. Here's my input code, you can see what's going on.
[module_Noahlsm_gridded_input.txt](https://github.com/user-attachments/files/17660752/module_Noahlsm_gridded_input.txt)
